src/Store.js

```diff
@@ -149,6 +149,10 @@ class Store {
     return this._replicationStatus
   }
 
+  setIdentity (identity) {
+    this._oplog.setIdentity(identity)
```

Comment:
I think we should need a `this.identity = identity` here as well, as `this.identity` is used elsewhere (when the db is dropped and when loading the db for example)

Reply:
Good catch, totally forgot to do that!